reconcile/slack_usergroups.py

```diff
@@ -793,9 +851,7 @@ def run(
         slack_map=slack_map,
         dry_run=dry_run,
     )
-    if error_occurred:
-        logging.error("Error(s) occurred.")
-        sys.exit(1)
+    return ExtendedEarlyExitRunnerResult(payload={}, applied_count=0)
```

Comment:
why do we use applied_count as 0?

Reply:
The blunt answer is: it is a required field and I did not care about this debugging info at the moment. Perhaps add it later?
Would require additional refactorings...

Reply:
`applied_count` is not debugging info, if it's not 0, next run won't use cache, until a run with nothing applied (applied_count = 0), this is the way to detect reconcile loop.

Reply:
My bad, I thought it was informational only.